src/org/ods/component/ScanWithSnykStage.groovy

```diff
@@ -0,0 +1,88 @@
+package org.ods.component
+
+import org.ods.services.SnykService
+
+class ScanWithSnykStage extends Stage {
+  protected String STAGE_NAME = 'Snyk Security Scan'
+  protected SnykService snyk
+
+  ScanWithSnykStage(def script, IContext context, Map config, SnykService snyk) {
+    super(script, context, config)
+    if (!config.containsKey('failOnVulnerabilities')) {
+      config.failOnVulnerabilities = context.failOnSnykScanVulnerabilities
```

Comment:
I like the test very much! Cool stuff! :-)
My coverage report reported that there is a missing test case for `!config.containsKey('failOnVulnerabilities')`

Reply:
Added as an issue: #238.